src/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -53,12 +53,25 @@
  * SAMRecords, and the header contains only sequence records.
  */
 @CommandLineProgramProperties(
-        usage = "Read fasta or fasta.gz containing reference sequences, and write as a SAM or BAM file with only sequence dictionary.\n",
-        usageShort = "Creates a SAM or BAM file from reference sequence in fasta format",
+        usage = CreateSequenceDictionary.USAGE_SUMMARY + CreateSequenceDictionary.USAGE_DETAILS,
+        usageShort = CreateSequenceDictionary.USAGE_SUMMARY,
         programGroup = Fasta.class
 )
 public class CreateSequenceDictionary extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Creates a SAM or BAM file from reference sequence in fasta format.  ";
+    static final String USAGE_DETAILS = "Writes a SAM/BAM file from a fasta-formated reference sequence.  " +
+            "The output SAM file contains a header, but no SAMRecords and the header contains only sequence records." +
```

Comment:
Hah, got it. This just makes a sequence dictionary, which I should have guessed from the name. This is just because some tools (like GATK) require a sequence dictionary to be provided separately with the reference sequence file. But it's not a real BAM file. This should all be rephrased to avoid confusing the heck out of users. (I know what this tool does and have used it myself, and yet was still super confused by that phrasing).